html-css/Task3/css/banner.css

```diff
@@ -0,0 +1,39 @@
+.banner {
+    color: #e6e6e6;
+    text-align: center;
+    font-family: 'Source Sans Pro Light';
+}
+
+.banner__title {
+    display: block;  
+    font-size: 30px;
+    letter-spacing: 5.28px; 
+    white-space: 1px;
+    background: #38373b; 
+    text-transform: uppercase;  
+    line-height: 34px; 
+}
+
+.banner__caption {
+    display: block;  
+    letter-spacing: 2.42px;
+    font-size: 20px;
+    text-transform: none;
+    background: #38373b; 
+}
+
+.banner__image {
+    height: 400px;
+    background: url("../images/Banner.png")  no-repeat;
+    background-position-x: 50%;
+}
+
+.banner__title_center {
```

Comment:
I think `banner__title_top` will be more suitable taking into account the fact that the other class is called `banner__title_bottom`